src/scalable_cuckoo_filter.rs

```diff
@@ -192,6 +193,33 @@ impl<T: Hash + ?Sized, H: Hasher + Clone, R: Rng> ScalableCuckooFilter<T, H, R>
         self.filters.iter().map(|f| f.bits()).sum()
     }
 
+    /// Returns the number of cuckoo filters.
+    pub fn num_filters(&self) -> usize {
+        self.filters.len()
+    }
+
+    /// Returns the number of cuckoo filters.
+    pub fn false_positive_probability(&self) -> f64 {
+        self.false_positive_probability
+    }
+
+    /// Returns the number of elements in each buckets.
+    pub fn bucket_size(&self) -> usize {
+        self.entries_per_bucket
+    }
+
+    /// Returns the number of kicks before the filter grows.
+    pub fn max_kicks(&self) -> usize {
+        self.max_kicks
+    }
+
+    /// Returns memory consume by the filter.
+    pub fn mem_usage(&self) -> usize {
+        self.bits() as usize
```

Comment:
This implementation appears to be incorrect at the following points:
- The unit of `self.bits()` is bits, while `mem::size_of()` is bytes
- The result of `mem::size_of::<ScalableCuckooFilter<&str>>` could be incorrect if the `H` and `R` type parameters are non default ones (`mem::size_of::<Self>()` should be used I think)
- `CuckooFilter` class contains a variable-sized value (i.e., `ExceptionalItems`), which is not accounted for in `mem::size_of()`

Reply:
Besides, I think that it is helpful for users if the method description contains the difference between this method and `bits()` method.

Reply:
[NOTE]
The size of Bits struct also should be considered.
But, as the limitation, this method does not account for the dynamic fileds of the hasher and rng.
(To be honest, I am not sure that providing this method, in addition to bits(), is really helpful for users.)

Reply:
I think bits will do.
Removed